README.adoc

```diff
@@ -165,7 +165,8 @@ $ make test-unit
 
 integration-tests::
 Integration tests demand more setup (i.e. the PostgreSQL DB must be already
-running) and probably time.
+running) and probably time. Please make sure you have installed psql and
+it is running on 5432 or if you have docker compose just do `docker-compose up -d`
```

Comment:
We also need to make sure the password is set just as in the docker-compose.yaml file. Therefore I'd say the only viable option is to use docker-compose instead of running it natively.